src/hooks/historikk/useDialogMedBehandlerHistorikk.ts

```diff
@@ -0,0 +1,59 @@
+import { HistorikkEvent } from "@/data/historikk/types/historikkTypes";
+import { useBehandlerdialogQuery } from "@/data/behandlerdialog/behandlerdialogQueryHooks";
+import {
+  MeldingDTO,
+  MeldingResponseDTO,
+} from "@/data/behandlerdialog/behandlerdialogTypes";
+import { meldingTypeTexts } from "@/data/behandlerdialog/behandlerdialogTexts";
+
+interface DialogMedBehandlerHistorikk {
+  isDialogMedBehandlerHistorikkLoading: boolean;
+  isDialogMedBehandlerHistorikkError: boolean;
+  dialogMedBehandlerHistorikk: HistorikkEvent[];
+}
```

Comment:
Trenger vi å ha prefix på alt her? 😊
```suggestion
interface DialogMedBehandlerHistorikk {
  isLoading: boolean;
  isError: boolean;
  events: HistorikkEvent[];
}
```
Så kan vi utenfor denne filen bruke det slik:
```
const dialogMedBehandlerHistorikk = useDialogMedBehandlerHistorikk();
dialogMedBehandlerHistorikk.isLoading
dialogMedBehandlerHistorikk.isError
dialogMedBehandlerHistorikk.events
```

Reply:
Fulgte i grunn bare tidligere mønster. Kan vel være for så vidt enig. Nå kan det se ut som at ingen av disse interfacene ser ut til å bli brukt utenfor filen da men.

Reply:
Enig i at vi kan endre det 👍🏼

Reply:
Da er det oppdatert. Ser jeg ikke helt forsto hvordan (selv om det sto i første kommentaren) dette ville påvirke bruken av det i `useHistorikk`, men tror det ble som ønsket.